app/assets/javascripts/social_networking/controllers/home-controller.js

```diff
@@ -61,7 +62,7 @@
   };
 
   HomeCtrl.prototype.retrieveFeed = function () {
-    var responsePromise = this._sharedResource.get('/social_networking/shared_items/participant/'+this._currentParticipantId+'/page/'+this.page);
+    var responsePromise = this._sharedResource.get('/social_networking/' + this._resource + '/participant/' + this._participantId + '/page/' + this.page);
```

Comment:
Are there circumstances where the resource changes to something other than a shared item?